user/user.go

```diff
@@ -60,6 +61,12 @@ func defaultRouter(env *env) *mux.Router {
 }
 
 func main() {
+	// Prometheus metrics
+	go func() {
+		http.Handle("/metrics", promhttp.Handler())
+		http.ListenAndServe(":2112", nil)
```

Comment:
Prometheus always on 2112? If not do we wanna make it a config field?

Reply:
Good shout, I guess we should do this for the service port too

Reply:
Will add a card